webpack/components/ProxmoxContainer/MountPoint.js

```diff
@@ -0,0 +1,58 @@
+import React, { useState, useEffect } from 'react';
+import PropTypes from 'prop-types';
```

Comment:
You may drop that, if you are not defining the component's Parameter-types and -defaults.
I am not sure if it is strictly necessary, but I guess it is best-practice.

Defining it helps with finding unexpected property-types/-formats coming in :grin: